feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+
+```bash
+simd version
+```
+
+You should see `0.44.0` printed to the console.
+
+## Configuration
+
+<!-- TODO: update tip to use `unsafe-reset-all` with better user experience -->
+
+::: tip
+If you have used `simd` before, you might already have a `.simapp` directory in your home directory. To keep the previous data, either save the directory to another location or use the `--home` flag and specify a different directory for each command in the following instructions. If you don't want to keep the previous data, remove the previous directory (`rm -rf ~/.simapp`).
+:::
+
+Run the following commands to configure the `simd` binary.
+
+Set the chain ID:
+
+```bash
+simd config chain-id demo
+```
+
+Set the [keyring backend](https://docs.cosmos.network/v0.42/run-node/keyring.html#the-test-backend):
+
+```bash
+simd config keyring-backend test
+```
+
+## Key Setup
+
+You'll have to create a few test keys for your users.
+
+Add a key for Alice:
+
+```bash
+simd keys add alice
+```
+
+Add a key for Bob:
+
+```bash
+simd keys add bob
+```
+
+And lastly, add a key for Charlie:
+
+```bash
+simd keys add charlie
+```
+
+If you'd like to see an overview of your keys, use:
+
+```bash
+simd keys list
+```
+
+In order to avoid having to copy-paste their address all the time, you should export them to variables that you can keep using during this tutorial:
+
+```bash
+export ALICE_KEY=$(simd keys show alice -a)
+export BOB_KEY=$(simd keys show bob -a)
+export CHARLIE_KEY=$(simd keys show charlie -a)
+```
+
+## Chain Setup
+
+The following commands set up a chain using the simulation application (`simapp`).
+
+Initialize the node:
+
+```bash
+simd init test --chain-id demo
+```
+
+Alice will be your validator. Add Alice and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account alice 5000000000stake --keyring-backend test
+```
+
+Add Bob and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account bob 2000kudos --keyring-backend test
+```
+
+Note that Bob only has `kudos` tokens and won't be able to pay for any fees that might be needed.
+
+Add Charlie and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account charlie 1000000stake --keyring-backend test
+```
+
+Generate a transaction to add Alice to the initial validator set:
+
+```bash
+simd gentx alice 1000000stake --chain-id demo
+```
+
+Add the validator transaction to the genesis file:
+
+```bash
+simd collect-gentxs
+```
+
+## Start Chain
+
+You are now ready to start a single node network on your local machine.
+
+Start the chain:
+
+```bash
+simd start
+```
+
+## Grant Allowance
+
+Before Bob can send `kudos` to Charlie, you'll need to setup an allowance for Bob, so that Charlie will pay for any gas fees the transaction might incur. You're not letting Alice pay the fees for Bob because Alice is the validator and would receive all those fees, which means you wouldn't be able to see the difference in her balance.
```

Comment:
```suggestion
Before Bob can send `kudos` to Charlie, you'll need to set up an allowance for Bob, so that Charlie will pay for any gas fees the transaction might incur. The exercise did not choose Alice as the granter because she is the validator too. Choosing Alice as both the validator and the granter would make the change in her balance as a result of her grantee's actions less evident.
```

Reply:
Not sure I agree with the wording here, but will make changes as I understand the way I phrased it might be confusing.